main.go

```diff
@@ -807,3 +902,21 @@ that will attach the file to your build as an artifact!`, logPth)
 		log.Warnf("Failed to export: BITRISE_XCODE_TEST_RESULT, error: %s", err)
 	}
 }
+
+func logDebugPretty(str string, v interface{}) {
```

Comment:
please separate these 2 func into a new subpackage temporary ( we are using similar function in more and more codes, so soon we need to put this into a whol new package, inspiration: https://github.com/bitrise-tools/xcode-project/blob/entitlements/pretty/pretty.go -> it has a nice interface: `pretty.Object(v)` )